File: rest/making-calls/example-4/example-4.5.x.cs
// Download the twilio-csharp library from twilio.com/docs/csharp/install
using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account;
using Twilio.Types;
using System.Collections.Generic;
class Example
{
  static void Main(string[] args)
  {
    // Find your Account Sid and Auth Token at twilio.com/console
    string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
    string authToken = "your_auth_token";
    TwilioClient.Init(accountSid, authToken);

    var url = new Uri("http://demo.twilio.com/docs/voice.xml");
    var to = new PhoneNumber("+14155551212");
    var from = new PhoneNumber("+18668675309");
    var method = "GET";
    var statusCallback = new Uri("https://www.myapp.com/events");
    var statusCallbackMethod = "POST";
    var statusCallbackEvents = new List<string>() { "initiated", "ringing", "answered", "completed" };

    var call = CallResource.Create(to,
                                   from,
                                   url: url,
                                   method: method,
                                   statusCallback: statusCallback,
                                   statusCallbackMethod: statusCallbackMethod,
                                   statusCallbackEvent: statusCallbackEvents);

    Console.WriteLine(call.Sid);
  }
}

File: rest/making-calls/example-4/example-4.5.x.php
<?php
// Get the PHP helper library from twilio.com/docs/php/install
require_once '/path/to/vendor/autoload.php'; // Loads the library
use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token = "your_auth_token";
$client = new Client($sid, $token);

$call = $client->calls->create(
    "+14155551212",
    "+18668675309",
    array(
        "url" => "http://demo.twilio.com/docs/voice.xml",
        "method" => "GET",
        "statusCallbackMethod" => "POST",
        "statusCallback" => "https://www.myapp.com/events",
        "statusCallbackEvent" => array(
            "initiated", "ringing", "answered", "completed"
        )
    )
);

echo $call->sid;
